packages/lesmis/src/api/api.js

```diff
@@ -12,6 +12,11 @@ const getApiUrl = () => {
   // if no env var, use sensible defaults based on the front end url
   const { hostname } = window.location; // eslint-disable-line no-undef
 
+  const { REACT_APP_LESMIS_API_URL } = process.env;
+  if (REACT_APP_LESMIS_API_URL) {
+    return REACT_APP_LESMIS_API_URL;
+  }
```

Comment:
It is very helpful to me because I'm coding in mac using a linux home server (everything is in my local network so it is secure). I also saw `REACT_APP_LESMIS_API_URL` in `.env.sample`. Hopefully it doesn't break anything?

Reply:
Actually it used to be this way, I'm not sure when it changed. Happy to have this back in place @billli0, but could you please move it above the comment that refers to "if no env var" which used to live after that check? That way the comment still makes sense too. Thanks!

Reply:
Make sense. 👍